src/components/Profile/LinkTo/LinkTo.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import { StyledLink } from './LinkTo.styled.js';
+
+const LinkTo = ({ to, title, className }) => {
```

Comment:
it's redundant too.

Reply:
delete & create common/common.sytled.js => StyledLink